themes/vscode/vscode.theme.sh

```diff
@@ -26,4 +26,4 @@ _omb_theme_vscode_initialize() {
     unset -f _omb_theme_vscode_initialize
 }
 _omb_theme_vscode_initialize
-export PROMPT_DIRTRIM=4
+PROMPT_DIRTRIM=4
```

Comment:
The export attribute is not needed because the variable `PROMPT_DIRTRIM` is basically a shell variable that is used to configure the current process of Bash but is not an environmental variable for external commands.